quiche/src/recovery/mod.rs

```diff
@@ -130,6 +130,13 @@ pub(crate) enum Recovery {
     GCongestion(GRecovery),
 }
 
+#[derive(Debug, Default, PartialEq)]
+pub struct OnAckReceivedResult {
+    pub lost_packets: usize,
+    pub lost_bytes: usize,
+    pub acked_bytes: usize,
+}
```

Comment:
In Rust, "Result" implies an <Ok, Err> enum by convention and makes it easy to tell if a function can fail. Can we rename this to something like `OnAckReceivedOutcome`

Reply:
Thanks for catching that.  Fixed.